source/sourcing_and_processing_data/processing.rst

```diff
@@ -1349,35 +1327,138 @@ It takes the following parameters:
     * Field
     * yes
 
-Simplify Geo Shape
-~~~~~~~~~~~~~~~~~~
+GeoJoin
+~~~~~~~
 
-Simplify a geo shape to reduce processing time and dataset size.
+This processor collection retrieves administrative divisions **Geo Shapes** from a key (postcode, county code, etc.). Each country has a dedicated processor and specific referentials.
+These referentials, which are referenced in the tables below, can be found in datasets available on https://public.opendatasoft.com.
+
+France
+^^^^^^
+
+====================  ==========================================================================================================
+Repository            Source
+====================  ==========================================================================================================
+postcode              `Carte des Codes Postaux <https://public.opendatasoft.com/explore/dataset/contour-des-codes-postaux>`_
+iris                  `Contours Iris - 2014 <https://public.opendatasoft.com/explore/dataset/contours-iris-2014>`_
+insee                 `Geofla® - Communes 2015 <https://public.opendatasoft.com/explore/dataset/geoflar-communes-2015>`_
```

Comment:
Shouldn't we give the link to Geofla 2016 instead? I believe Geofla - Communes 2016 is now available.